pyinstaller/electron/main.js

```diff
@@ -329,8 +329,7 @@ function updatingLoaderMsg(msg) {
 }
 
 function hasSuccessfullyStarted(logs) {
-  return logs.toString().includes('  * Running on http')
-  //return logs.toString().includes('Serving Flask app "cryptoadvance.specter.server"')
+  return logs.toString().includes('Configuration')
```

Comment:
That sounds very unspecific. Why did that original term disappeared? I found it quite appropriate.

Reply:
Changed.